### ISSUE TITLE
Node version problem

### ISSUE THREAD

**Issue by ubombar** [ghivert/collectible-card-game-daar#1]
Hello,
I am trying to spin up the repository in M2 Mac. But I encountered a problem when I run the command ``yarn``. It gave me the following output:
> $ yarn         
yarn install v1.22.19
[1/4] 🔍  Resolving packages...
success Already up-to-date.
$ yarn --cwd contracts install && yarn --cwd frontend install
yarn install v1.22.19
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
warning Pattern ["hardhat-deploy-ethers@^0.3.0-beta.13"] is trying to unpack in the same destination "/Users/bombar/Library/Caches/Yarn/v6/npm-@nomiclabs-hardhat-ethers-0.3.0-beta.13-b96086ff768ddf69928984d5eb0a8d78cfca9366-integrity/node_modules/@nomiclabs/hardhat-ethers" as pattern ["@nomiclabs/hardhat-ethers@npm:hardhat-deploy-ethers"]. This could result in non-deterministic behavior, skipping.
error hardhat@2.12.0: The engine "node" is incompatible with this module. Expected version "^14.0.0 || ^16.0.0 || ^18.0.0". Got "20.8.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

I am guessing this error is because of a mismatch in my node version and the library ``hardhat``. I will try to test this with another node installation with ``nvm`` so others encountering the same problem can refer to this post.

Before I finish here are my ``node``, ``yarn``, and ``npm`` versions:
> $ yarn --version
1.22.19
$ npm --version
10.1.0
$ node --version
v18.17.0

**Comment by ghivert** [comment]
You're correct, the Node engine should be at 18. `nvm` and `nvm install/use 18` should be enough! Thanks for pointing this out, I'll update the repo.

**Comment by ubombar** [comment]
I am not sure why my node gave me this error since it is version 18.*. But I tried to reinstall it and now works!
